src/PIL/GimpGradientFile.py

```diff
@@ -64,9 +65,22 @@ def sphere_decreasing(middle, pos):
 
 
 class GradientFile:
-    gradient = None
-
-    def getpalette(self, entries=256):
+    gradient: (
+        list[
+            tuple[
+                float,
+                float,
+                float,
+                list[float],
+                list[float],
+                Callable[[float, float], float],
+            ]
+        ]
+        | None
+    ) = None
```

Comment:
We also have this long type def in `GimpGradientFile`, is it worth assigning it to something and re-using it?

Reply:
I've realised we can just re-use the other variable. I've pushed a commit.